website/frontend/templates/docs/plugin-api.html

```diff
@@ -71,6 +75,36 @@ <h1 id="file_formats">File Formats</h1>
 
       register_format(MyFile)</pre>
 
+    <p>
+    Variables explanation:
+    <ul>
+      <li>"PLUGIN_DESCRIPTION" should be as simple as possible, while still describing the main function.</li>
+      <li>"PLUGIN_VERSION" should be filled with the version of Plugin.</li>
+      <li>"PLUGIN_API_VERSIONS" should be set to the version of Picard this plugin to run with.</li>
+      <li>"PLUGIN_LICENSE" should be set with the license name of the plugin. You can choose on your desired license's abbreviation in <a href="https://spdx.org/licenses/">this license database</a>.</li>
+      <li>"PLUGIN_LICENSE_URL" should be set with the license url.</li>
+    </ul>
+    </p>
+    <p>
+    PLUGIN_API_VERSIONS available:</p>
+    <p>v1.0</p>
+    <ul>
+        <li>0.15.1</li> 
+        <li>0.16.0</li>
+        <li>1.0.0</li>
+        <li>1.1.0</li>
+        <li>1.2.0</li>
+        <li>1.3.0</li>
+        <li>1.4.0</li>
+    </ul>
+    </p>
```

Comment:
This closing `</p>` doesn't have an opening one. I think maybe you meant `<p>v1.0</p>` to be `<p>v1.0`?